### PR TITLE
fix_multiple_browsers_issue

### PR DESCRIPTION
**Pull request by artati**

### CODE REVIEW

File: src/test/java/helper/Base.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Base {
+
+        public static WebDriver driver=new ChromeDriver();
+        public static void closeSession() {
+                driver.quit();
+        }
+        public static void init(String url){
```
Comment (GheorgheSacultanu) [inline src/test/java/helper/Base.java]: Maybe should be better to have first the init session then the close session?

File: src/test/java/helper/Base.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package helper;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Base {
```
Comment (GheorgheSacultanu) [inline src/test/java/helper/Base.java]: Maybe we can have a better class name. Smth like ,,Initialization''

**Review by GheorgheSacultanu — COMMENTED** [review]
Need rework

**Review by GheorgheSacultanu — CHANGES_REQUESTED** [review]
Class Name + first should be init method and after close method.

File: src/test/java/helper/Base.java (hunk 2 above)
Comment (artati) [reply src/test/java/helper/Base.java]: Fixed

File: src/test/java/helper/Base.java (hunk 1 above)
Comment (artati) [reply src/test/java/helper/Base.java]: done

**Review by GheorgheSacultanu — APPROVED** [review]

**Pull request merged**